docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/configure-shibboleth-saml/about-group-permissions.md

```diff
@@ -2,9 +2,11 @@
 title: Group Permissions with Shibboleth and OpenLDAP
 ---
 
-This page provides background information and context for Rancher users who intend to set up the Shibboleth authentication provider in Rancher.
+<head> 
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/pages-for-subheaders/configure-shibboleth-saml"/>
```

Comment:
```suggestion
  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/configure-shibboleth-saml/about-group-permissions"/>
```

Should the canonical link point to `about-group-permissions` instead (self reference in this file)?

Reply:
You're right -- that was a mistake. I'll push a fix.

Reply:
Fixed in e9c99fe